x/src/cargo.rs

```diff
@@ -75,6 +77,25 @@ impl Cargo {
         self
     }
 
+    // Add a series of arguments to the x command.
+    pub fn args<I, S>(&mut self, args: I) -> &mut Self
+    where
+        I: IntoIterator<Item = S>,
+        S: AsRef<OsStr>,
```

Comment:
You can inline these type params if you want:

```
pub fn args(&mut self, args: impl IntoIterator<Item = impl AsRef<OsStr>>)
```

I personally like this pattern but it's really a matter of taste.

Reply:
thanks, I'll try that out in the future.   Seems it may be a tad more readable - but still getting my legs here, so unsure.

Reply:
plus the project already seem standardized on the where syntax.